### PR TITLE
[ENGINEERS-910] - Increased hard retries to verify pickup points 

### PR DESCRIPTION
**Pull request by shashidhar85**
**Issue**: We noticed that getStores graphql api is taking some time to give updated stores in response.

**Fix** :
- Increased delay time between retries to 10 sec from 4 sec
- Increased hard retries to 3 from 1

JIRA Ticket - [ENGINEERS-910](https://vtex-dev.atlassian.net/browse/ENGINEERS-910)

### CODE REVIEW

File: cy-runner.yml [hunk 1]
```diff
@@ -7,13 +6,8 @@ base:
     account: productusqa
     id: 2001459804
     domain: myvtex.com
+    urlExternalSeller: https://productusqaseller.myvtex.com
```
Comment (Syed-Vtex) [inline cy-runner.yml:9]: does cy-runner throwing any error when this key is not present
@shashidhar85

**Review by Syed-Vtex — CHANGES_REQUESTED** [review]

File: cy-runner.yml (hunk 1 above)
Comment (shashidhar85) [reply cy-runner.yml:9]: no

File: cy-runner.yml [hunk 2]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (Syed-Vtex) [inline cy-runner.yml:1]: why this been removed

File: cy-runner.yml [hunk 3]
```diff
@@ -44,7 +37,6 @@ base:
     pageLoadTimeout: 45000
     browser: chrome
     chromeWebSecurity: false
-    sorry: false
```
Comment (Syed-Vtex) [inline cy-runner.yml:47]: same here

**Review by Syed-Vtex — APPROVED** [review]

**Pull request merged**